CONTRIBUTE.md

```diff
@@ -0,0 +1,45 @@
+## Setup panel
+
+1. clone the plugin to `%installationpath%/Grafana/grafana/plugins`
+2. Install dependencies `npm i`
+3. Build project `yarn build`
+
+## Setup demo env with minikube and (altered) sockshop
+
+### Minikube
+
+```
+minikube start --memory 8192 --cpus 4
+```
+
+#### Prometheus
+
+```BASH
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+helm repo update
+```
+
+#### cAdvisor
+```BASH
+helm repo add ckotzbauer https://ckotzbauer.github.io/helm-charts
+helm install my-cadvisor ckotzbauer/cadvisor --version 1.2.4
+```
+
+#### Sockshop
+
+1. clone [Sock Shop](https://github.com/microservices-demo/microservices-demo)
+2. replace `%installationpath%Sockshop/microservices-demo/deploy/kubernetes/complete-demo.yaml` with `%installationpath%/novatec-k8s-panel/complete-demo.yaml`. You can find the [images on docker hub](https://hub.docker.com/u/thesisphilip). You might need to add them manually.
```

Comment:
is the complete-demo file updated to the one that we are using now ? because the first one had another apiVersion

Reply:
You are right in the github repo is still the old version of `complete-demo.yaml`. Should I update the file in this PR as well?

Reply:
yes I think it would be better

Reply:
✅